wscript

```diff
@@ -172,11 +175,12 @@ def build(bld):
     robot_dart_magnum_srcs = " ".join(magnum_files)
 
     libs = 'BOOST EIGEN DART PTHREAD'
-
+    defines = ["ROBOT_DART_PREFIX=\"" + bld.env['PREFIX'] + "\""]
```

Comment:
We need to create a `config.hpp` that gets installed with the library and contains these directories for people trying to use the installed library. Even if they do not really need it, it's good have them automatically generated and saved somewhere for reference.

Reply:
OK. I can do this.